src/test/java/org/commcare/backend/session/test/ChildModuleNavigationTests.java

```diff
@@ -24,12 +27,20 @@ public void testNeedsCommandFirst() throws Exception {
 
         // since there are two entries registered under 'parent-module' that
         // need different case data, we need to select the entry before the case
-        assertEquals(session.getNeededData(), SessionFrame.STATE_COMMAND_ID);
+        assertEquals(SessionFrame.STATE_COMMAND_ID, session.getNeededData());
```

Comment:
switching the order of these so that it's `(expected, actual)`, as the method specification dictates